webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -150,7 +167,23 @@ class NodeGroupForm(
        OnLoad(JsRaw("$('#GroupTabs').tabs( {active : 0 });" ))
      )
 
-     (
+    (nodeGroup match {
+      case Left(target) => showFormTarget(target)
+      case Right(group) if(group.isSystem) => showFormTarget(GroupTarget(group.id))
+      case Right(group) =>
+        showFormNodeGroup(group)
+    })(html)
+  }
+
+  private[this] def showRulesForTarget(target: SimpleTarget): NodeSeq = {
+    val noDisplay = DisplayColumn.Force(false)
+    val cmp = new RuleGrid("rules_for_current_group", None, false, None, noDisplay, noDisplay)
+    val rules = dependencyService.targetDependencies(target).map( _.rules.toSeq.filter(!_.isSystem)).toOption
+    cmp.rulesGridWithUpdatedInfo(rules, false, false)
+}
+
+  private[this] def showFormNodeGroup(nodeGroup: NodeGroup): CssSel = {
+(
```

Comment:
Parenthesis should be idented at the method level

Reply:
done